docs/content/concepts/assets/asset-checks/define-execute-asset-checks.mdx

```diff
@@ -157,6 +160,18 @@ defs = Definitions(
 )
 ```
 
+### TODO: multi_asset_check
```

Comment:
"Defining multiple checks separately from an asset" (or assets)?

Reply:
yep, or it could be a subsection of ### Defining checks separately from assets

Reply:
Oooh I like that better, in it goes